run_tests.sh

```diff
@@ -0,0 +1,46 @@
+TESTS_PATH=""
```

Comment:
```suggestion
#!/usr/bin/env bash
TESTS_PATH=""
```

And then there's no need to use `bash run_tests.sh`, but `./run_tests.sh`.

Reply:
In fact, it needs to be `bash run_tests.sh`, so that running inside Visual Studio Code terminal it doesn't launch a new terminal instance like with `./run_tests.sh`. Not sure if it applies for Linux, but this is the case for Windows

Reply:
Also got this error: `'.' is not recognized as an internal or external command,
operable program or batch file.`

Reply:
It doesn't apply to Linux/macOS, so I'd prefer to at least have this header in the file.